stubs/aiofiles/@tests/stubtest_allowlist.txt

```diff
@@ -9,6 +9,15 @@ aiofiles.threadpool.binary.AsyncBufferedIOBase.readable
 aiofiles.threadpool.binary.AsyncBufferedIOBase.seekable
 aiofiles.threadpool.binary.AsyncBufferedIOBase.tell
 aiofiles.threadpool.binary.AsyncBufferedIOBase.writable
+aiofiles.threadpool.binary.AsyncIndirectBufferedIOBase.close
```

Comment:
All of the allowlist are copy pasting the non-indirect version above it. This version introduces indirect variants of most of IO classes that need exception for same reason (at runtime they delegate with *args/**kwargs).